packages/airnode-deployer/src/cli/index.ts

```diff
@@ -95,21 +91,39 @@ yargs(hideBin(process.argv))
       'cloud-provider': {
         alias: 'c',
         description: 'Cloud provider',
-        type: 'string',
+        choices: ['aws', 'gcp'] as const,
       },
       region: {
         alias: 'e',
         description: 'Region',
         type: 'string',
       },
+      'project-id': {
+        alias: 'p',
+        description: 'Project ID (GCP only)',
+        type: 'string',
+      },
     },
     async (args) => {
       logger.debugMode(args.debug as boolean);
       logger.debug(`Running command ${args._[0]} with arguments ${longArguments(args)}`);
       const receiptRemove = !!args.receipt;
-      const descriptiveArgs = ['airnodeAddressShort', 'stage', 'cloudProvider', 'region'];
-      const descriptiveArgsProvided = intersection(descriptiveArgs, keys(args));
-      const descriptiveArgsMissing = difference(descriptiveArgs, descriptiveArgsProvided);
+      const descriptiveArgsCommon = ['airnode-address-short', 'stage', 'cloud-provider', 'region'];
+      const descriptiveArgsCloud = {
+        aws: descriptiveArgsCommon,
+        gcp: [...descriptiveArgsCommon, 'project-id'],
+      };
+      const descriptiveArgsAll = uniq(
+        Object.values(descriptiveArgsCloud).reduce((result, array) => [...result, ...array])
+      );
+      const argsProvided = intersection([...descriptiveArgsAll, 'receipt'], keys(args));
```

Comment:
do we need this? can we use `descriptiveArgsProvided`

Reply:
Those are missing the `receipt` argument. If users don't provide any argument I want to let them know that they have to either provide a receipt file with `--receipt` or describe the deployment with other arguments.